tools/oesign/oesign.c

```diff
@@ -1,13 +1,15 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#define __STDC_WANT_LIB_EXT1__ 1
```

Comment:
Does GCC even support this? You're also not using the extended "safe" versions of the functions, so this seems unnecessary.

Reply:
Remove it per comment.